docs/WPF/02-Initializing.md

```diff
@@ -51,7 +51,7 @@ The **CreateShell** method allows a developer to specify the top-level window fo
 
 An example of using the **ServiceLocator** to resolve the shell object is shown in the following code example.
 
-```
+```cs
```

Comment:
yes. we were handling code references in the docs three different ways. the newest docs were already specifying code language, the older docs either indented like on StackOverflow or didn't specify the language type like this.